Packs/CybelAngel/Integrations/CybelAngelEventCollector/CybelAngelEventCollector_description.md

```diff
@@ -0,0 +1,20 @@
+CybelAngel is a cybersecurity firm specializing in external attack surface protection and management. 
```

Comment:
shouldnt this description be the same as is in the readme? There should be details about what is being collected in this collector

Reply:
change it to 'CybelAngel Event Collector collects reports from the CybelAngel platform, which specializes in external attack surface protection and management.' ?? or change both to 'CybelAngel Event Collector receives reports from the CybelAngel platform, which specializes in external attack surface protection and management.' ?

Reply:
@tschanfeld The second one